expression/builtin_other.go

```diff
@@ -797,6 +797,19 @@ func (b *builtinValuesIntSig) evalInt(_ chunk.Row) (int64, bool, error) {
 		if row.IsNull(b.offset) {
 			return 0, true, nil
 		}
+		// For BinaryLiteral, see issue #15310
+		val := row.GetRaw(b.offset)
+		if len(val) > 8 {
+			return 0, true, errors.New("Session current insert values is too long")
+		}
+		if len(val) < 8 {
```

Comment:
Regarding it as a binary number when the length is less than `8` seems tricky, but I don't have any other idea which can fix this issue easily, do you have any suggestions? @SunRunAway

Reply:
How about checking the `FieldType` of args[0] whether is `mysql.TypeBit`?

Reply:
I have a try, but the function doesn't have args...  len(args) == 0.  @SunRunAway